.travis.yml

```diff
@@ -4,6 +4,7 @@ node_js:
   - "0.10"
 before_script:
   - cp config.js.example config.js
+  - npm run nsp
```

Comment:
I thought of that, @astorije (doing it `after_script`). The reason I put it *before* is that I still think builds should break if `nsp` fails. Otherwise, we'd never see those scary red icons when `nsp` returned &ne; 0&hellip;

Vulnerabilities appear because of new security breaches being reported, or new insecure dependencies being introduced in the project. In the first case, the author of that PR simply has to restart the latest build for `master` (or for any other branch that was OK), which surely will fail too, to &ldquo;prove his/her innocence&rdquo;. In the other case, the build for that PR fails&hellip; and rightly so.